VideoWeb/VideoWeb.Common/VideoWeb.Common.csproj

```diff
@@ -4,7 +4,7 @@
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="BookingsApi.Client" Version="2.3.8" />
+    <PackageReference Include="BookingsApi.Client" Version="2.3.10-pr-0893-0024" />
```

Comment:
just a reminder to update before merging

Reply:
done